vllm/engine/async_llm_engine.py

```diff
@@ -400,13 +402,15 @@ async def add_request(
             prompt=prompt,
             sampling_params=sampling_params,
             prompt_token_ids=prompt_token_ids,
-            arrival_time=arrival_time)
+            arrival_time=arrival_time,
+            prefix_pos=prefix_pos)
 
         return stream
 
     async def generate(
         self,
         prompt: Optional[str],
+        prefix_pos: Optional[int],
```

Comment:
Hi, @caoshiyi.

This function might be used by others who use the engine in their project directly.
To ensure backward compatibility, I thought it is better to add new parameters at the end, just like `AsyncLLMEngine.add_request` and `LLMEngine.add_request`.